WoWPro_Leveling/Wrath/Horde/WOTLK_Borean_Tundra.lua

```diff
@@ -467,19 +450,16 @@ A Cracking the Code |QID|11931|PRE|11919|N|From Raelorasz.|M|33.31,34.53|
 N Drake Hunt |QID|11931|N|Drake Hunt is now a repeatable daily quest. You may pick it up again and if you wish and do it again after the next step.|
 C Cracking the Code |QID|11931|N|Kill Inquisitor Caleras to the north and loot the Azure Codex. Kill Coldarra Spellbinders around him until you get 3 Shimmering Runes.|M|32.77,27.84|
 T Cracking the Code |QID|11931|N|To Raelorasz.|M|33.31,34.53|
-;Change FP, not picked up outpost yet
 F Amber Ledge|QID|12486|N|Fly back to Amber Ledge.|M|33.12,34.46|
 R Bor'gorok Outpost|QID|12486|N|Ride to Bor'gorok Outpost.|M|49.62,10.58|
 f Bor'gorok Outpost|QID|12486|N|Get the flightpoint as Kimbiza|M|49.59,10.99|
 A Report to Steeljaw's Caravan |QID|11591|LEAD|11594|N|From Overlord Bor'gorok.|M|49.62,10.58|
 T To Bor'gorok Outpost, Quickly! |QID|12486|M|50.3,9.7|N|To Spirit Talker Snarlfang.|
 A The Sky Will Know |QID|11624|M|50.3,9.7|N|From Spirit Talker Snarlfang.|
 A King Mrgl-Mrgl |QID|11702|N|From Supply Master Taz'ishi.|M|48.96,10.26|
-;fix M
 T The Sky Will Know |QID|11624|N|To Imperean.|M|46.54,9.27|
 A Boiling Point |QID|11627|PRE|11624|M|46.6,9.3|N|From Imperean.|
-;QO
-N Churn has submitted|QID|11627|QO|2|M|46,13|
+C Churn has submitted|QID|11627|QO|2|M|46,13|
```

Comment:
Here, both nasty no decimal coordinates, ill defined summary and no N tag.  Noting that Churn is immune to frost damage is an important detail to poor mages.